client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/ContactDateCard/InteractionCard/ExcelUploader/ContactUploader.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import axios from 'Utils/axios';
+import {Button, Typography} from '@material-ui/core';
+
+import ExcelLogo from './ExcelLogo.png';
```

Comment:
shouldn't this be in assets?

Reply:
assets is in public folder which cant but accessed from src

Reply:
ill add one in utils maybe, though it seems more comfortable to see the image in the place its used

Reply:
ill ask the team